CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#1042](https://github.com/nix-rust/nix/pull/1042))
 - Add `sched_yield`
   ([#1050](https://github.com/nix-rust/nix/pull/1050))
+- Add `unlinkat`
+  ([#TBD](https://github.com/nix-rust/nix/pull/1058))
```

Comment:
Don't forget to replace the other TBD.

Reply:
I see. Expecting either is the better option but it's not clear to me the expect way to account for this when assigning a string to expected for the match. Any pointer?